composer.json

```diff
@@ -21,15 +21,18 @@
         "codeception/codeception": "^4.0"
     },
     "require-dev": {
-        "doctrine/orm": "^2",
         "doctrine/annotations": "^1",
         "doctrine/data-fixtures": "^1",
-        "ramsey/uuid-doctrine": "^1.5"
+        "doctrine/orm": "^2",
+        "phpstan/phpstan": "^0.12.93",
+        "ramsey/uuid-doctrine": "^1.5",
+        "symfony/cache": "^5.3"
```

Comment:
surely that fixes continuous integration error, otherwise allow `symfony/cache` 4.4 as well

Reply:
@olexp this also seems to be necessary for your PR (#45).

Reply:
Looks like choice of cache was left to developer. doctrine/cache was already installed with other dependencies. But I have to downgrade it to 1.11.3 because 2.X has needed adapters removed. So, choice of symfony/cache seems reasonable.